test/spec/sanity-checks.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env mocha */
+
+var request = require('supertest')
+var assert = require('assert')
+var path = require('path')
+var fs = require('fs')
+
+/**
+ * Basic sanity checks on the dev server
+ */
+describe('The prototype kit', function () {
+  var app
+  var server
+
+  before(function () {
+    app = require('../../server')
+    server = app.listen(3000)
```

Comment:
In my initial research (see the thread above on github) it wasn't closing, and that was causing problems